content/en/tutorials/lambda-ecr-container-images/index.md

```diff
@@ -128,67 +135,78 @@ $ awslocal ecr describe-images --repository-name localstack-lambda-container-ima
+- `function-name`: Specifies the name of the Lambda function you want to create.
+- `runtime`: Defines the runtime environment for the Lambda function. In this case, it's specified as provided, indicating that the container image will provide the runtime.
+- `role`: Sets the IAM role ARN that the Lambda function should assume. In the example, a mock role ARN is used. For an actual role, please refer to the [IAM documentation]({{< ref "iam" >}}).
 
-Let us now invoke the Lambda function using the `invoke` command:
+To invoke the Lambda function, you can use the `invoke` command:
 
 {{< command >}}
 $ awslocal lambda invoke --function-name localstack-lambda-container-image /tmp/lambda.out
 {
     "StatusCode": 200,
-    "LogResult": "",
     "ExecutedVersion": "$LATEST"
 }
 {{< / command >}}
 
-The logs of the Lambda invocation should be visible in the LocalStack container output (with `DEBUG=1` enabled):
+The command above will execute the Lambda function locally within the LocalStack environment. The response will include the StatusCode, LogResult, and ExecutedVersion. You can find the logs of the Lambda invocation in the LocalStack container output, precisely when `DEBUG=1` is enabled.
 
 {{< command >}}
-Starting XRay server loop on UDP port 2000
-Starting DNS server loop on UDP port 53
------
 Hello from LocalStack Lambda container image!
 {{< / command >}}
 
 ## Conclusion
 
-With the Lambda container image support, you can use Docker to package your custom code and dependencies for Lambda functions. LocalStack allows you to package, deploy, and invoke Lambda functions locally. Using LocalStack, you can develop, debug, and test your Lambda functions in conjunction with a wide range of AWS services. Check out [Lambda Hot Reloading]({{< ref "hot-reloading" >}}) and [Lambda Hot Reloading]({{< ref "debugging" >}}) for advanced usage patterns.
+In conclusion, the Lambda container image support enables you to use Docker to package your custom code and dependencies for Lambda functions. With the help of LocalStack, you can seamlessly package, deploy, and invoke Lambda functions locally. It empowers you to develop, debug, and test your Lambda functions with a wide range of AWS services. For more advanced usage patterns, you can explore features like [Lambda Hot Reloading]({{< ref "hot-reloading" >}}) and [Lambda Debugging]({{< ref "debugging" >}}).
 
-The code for this tutorial (including a `Makefile` to execute it step-by-step) can be found in our [LocalStack Pro samples over GitHub](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-container-image).
+To further explore and experiment with the concepts covered in this tutorial, you can access the code and accompanying `Makefile` on our [LocalStack Pro samples over GitHub](https://github.com/localstack/localstack-pro-samples/tree/master/lambda-container-image).
```

Comment:
The changes here should be reflected in the LS pro sample: https://github.com/localstack/localstack-pro-samples/tree/master/lambda-container-image